pypit/arsave.py

```diff
@@ -73,12 +74,12 @@ def save_extraction(slf, sciext, scidx, scierr=None, filename="temp.fits", frame
     hdulist[0].header[hdrname] = (slf._fitsdict['filename'][scidx[0]], 'ARMED: Name of file that was extracted'.format(frametype))
     hdulist[0].header["FRAMETYP"] = (frametype, 'ARMED: extraction frame')
     hdulist[0].header["NUMORDS"] = (sciext.shape[1], 'ARMED: Number of orders extracted')
-    hdulist[0].header["PIXSIZE"] = (slf._argflag['reduce']['pixelsize'], 'ARMED: The size of each sampled pixel (km/s)')
+    hdulist[0].header["PIXSIZE"] = (settings.argflag['reduce']['pixelsize'], 'ARMED: The size of each sampled pixel (km/s)')
     # Loop through all orders and write the wavelength into the header
     if wave is not None:
         for i in range(sciext.shape[1]):
             hdrname = "CDELT{0:03d}".format(i+1)
-            hdulist[0].header[hdrname] = (np.log10(1.0 + slf._argflag['reduce']['pixelsize']/299792.458), 'ARMED: log10(1+pixsize/c)'.format(frametype))
+            hdulist[0].header[hdrname] = (np.log10(1.0 + settings.argflag['reduce']['pixelsize']/299792.458), 'ARMED: log10(1+pixsize/c)'.format(frametype))
```

Comment:
Indeed, this was used in a previous version of the ARMED code. I am planning a code cleanup as an upcoming PR to remove all deprecated functions, and remove the `dispdir` and `dispaxis` variables throughout the code, which are also no longer used/needed.